lib/client.js

```diff
@@ -124,7 +124,11 @@ function isValidEvent(client, event) {
 function triggerEvent(client, eventName, data) {
   if (!client._messageHandlers[eventName]) { return false; }
   client._messageHandlers[eventName].forEach(function(handler) {
-    handler(data);
+    if (Array.isArray(data)) {
+      handler.apply(undefined, data);
```

Comment:
If we are using ES6, then you can simply make this `handler(...data)`

Also, is there any difference between passing `undefined` or `null` as the first argument to `apply`?

Reply:
We are not useing es6 in the sdk.

Yes there is a difference., in the first `this` will be `undefined`, in the latter it will be `null`